### PR TITLE
[Done] INBOX: Truncate URL links to 50chars #169 (#178)

### PR DESCRIPTION
**Pull request by chrisns**
* [Done] INBOX: Truncate URL links to 50chars #169

* implemented edit URL functionality

Co-authored-by: rahulbodara <rb.globalia@gmail.com>

### CODE REVIEW

**Comment by mrjamieeast** [comment]
This does not work for me
Screenshot 2022-05-07 at 08 17 02

**Comment by chrisns** [comment]
@mrjamieeast have you retested this?

**Comment by mrjamieeast** [comment]
This now works for me

File: cypress/integration/podcasts.spec.js [hunk 1]
```diff
@@ -151,6 +151,29 @@ describe("newsdesk logged in", () => {
...
+  it("should add links in items", () => {
+    cy.contains("dev sandbox").click()
+    const link =
+      "https://twitter.com/PoliticusSarah/status/1520759587128979458?s=20&t=-ZSrWH2DIXO97dJgtoy46Q"
+    cy.get("section[slotno=7] textarea").type(`${link}{enter}`, {
+      force: true,
+    })
+    cy.get("section[slotno=7]").should(
+      "contain",
+      link.length > 50 ? link.slice(0, 50) + "..." : link,
```
Comment (chrisns) [inline cypress/integration/podcasts.spec.js:164]: the link is a fixed variable, so i'm not sure why you're doing any conditional here
better would be
```suggestion
      "https://twitter.com/PoliticusSarah/status/1520759..."
```

File: src/components/Inbox.vue [hunk 2]
```diff
@@ -12,7 +12,6 @@
 
 <script lang="ts" setup>
 import { watch } from "vue"
-import LinkifyIt from "linkify-it"
```
Comment (chrisns) [inline src/components/Inbox.vue:15]: have we removed LinkifyIt now? if so please `npm uninstall --save LinkifyIt`

File: cypress/integration/podcasts.spec.js [hunk 3]
```diff
@@ -151,6 +151,29 @@ describe("newsdesk logged in", () => {
...
+    cy.contains("dev sandbox").click()
+    const link =
+      "https://twitter.com/PoliticusSarah/status/1520759587128979458?s=20&t=-ZSrWH2DIXO97dJgtoy46Q"
+    cy.get("section[slotno=7] textarea").type(`${link}{enter}`, {
+      force: true,
+    })
+    cy.get("section[slotno=7]").should(
+      "contain",
+      "https://twitter.com/PoliticusSarah/status/15207595...",
+    )
```
[the remark this answers is not in the document]
Comment (bhavinsen) [reply cypress/integration/podcasts.spec.js]: I have updated the test case with `href` 
Please refer -> https://i.imgur.com/uwQXjnq.png

**Pull request merged**